src/pilot/assign_role.py

```diff
@@ -735,7 +735,7 @@ def configure_raid(ironic_client, node_uuid, role, os_volume_size_gb,
     when RAID configuration failed and return False. Further testing
     should uncover interesting error conditions.'''
 
-    if get_raid_controller_ids(drac_client) is None:
+    if not get_raid_controller_ids(drac_client):
         LOG.warning("No RAID controller is present.  Skipping RAID "
```

Comment:
Can be set as "Found no RAID controller" just to keep consistent throughout the module.
Please refer to line number 253.
LOG level warning or critical? May need to confirm with Aija once.

Reply:
resolved asdiscussed.